### PR TITLE
test: using config file to run daemon test

### PR DESCRIPTION
**Pull request by zhuangqh**
*background*: some env may depend on some
specific config options.

Signed-off-by: zhuangqh <zhuangqhc@gmail.com>

### CODE REVIEW

**Comment by fuweid** [comment]
@zhuangqh I trigger the CI to rerun it.

**Comment by fuweid** [comment]
```
go test -c \
	-o bin/pouchd-integration-test github.com/alibaba/pouch/test
# github.com/alibaba/pouch/test [github.com/alibaba/pouch/test.test]
test/cli_logs_test.go:254:34: cannot use "--log-opt" (type string) as type map[string]interface {} in argument to StartDefaultDaemon
Makefile:215: recipe for target 'build-integration-test' failed
make: *** [build-integration-test] Error 2
The command "./travis_ci.sh integrationtest 4 0" exited with 2.
```

ping @zhuangqh

**Comment by fuweid** [comment]
ping @zhuangqh any update?

File: test/cli_logs_test.go [hunk 1]
```diff
@@ -251,7 +251,9 @@ func (suite *PouchLogsSuite) TestSetLogPathInDaemon(c *check.C) {
 	logRootDir := "/tmp/TestSetLogsPathInDaemon"
 	defer os.RemoveAll(logRootDir)
 
-	dcfg, err := StartDefaultDaemon("--log-opt", fmt.Sprintf("root-dir=%s", logRootDir))
+	dcfg, err := StartDefaultDaemon(map[string]interface{}{
+		"--log-opt": fmt.Sprintf("root-dir=%s", logRootDir),
```
Comment (cxz66666) [inline test/cli_logs_test.go:255]: This need use args instead of config field? Or use `log-opt` instead of `--log-opt`? I'm not sure 😢

**Review by cxz66666 — CHANGES_REQUESTED** [review]

**Comment by cxz66666** [comment]
Thank you for your contribution! because you don't response for a long time, I will close this PR and reopen https://github.com/alibaba/pouch/pull/3036.

**Pull request closed without merging**